pkg/config/v2/config_writer_test.go

```diff
@@ -914,7 +917,7 @@ func TestWriteConfigs(t *testing.T) {
 
 	for _, tc := range tests {
 		t.Run(tc.name, func(t *testing.T) {
-			fs := afero.NewMemMapFs()
+			fs := util.TempFs(t)
```

Comment:
We do get an error: 'AppData\Local\Temp\TestWriteConfigsSettings_2.0_schema_write_sanitizes_names1064148250\001\test\project\builtin:alerting-profile: The directory name is invalid.'

It does not tell us exactly why it is invalid, but that's a limitation we have to live with. Otherwise we would have to implement and check each single rule for each OS

Reply:
that's perfectly fine I'd say :) thx for doublechecking